models/Profile/Event.js

```diff
@@ -34,6 +34,11 @@ const EventSchema = new Schema({
   price: {
     startingFrom: Number,
   },
+  userStatus: {
+    type: String,
+    enum: ["speaking", "not-speaking"],
+  },
+  speakingTopic: String,
```

Comment:
This needs a max, maybe 256?

Reply:
Here:
```
speakingTopic: {
    type: String,
    max: 256
  }
```

Reply:
Please make the change in your PR, it is easier to review

Reply:
You marked this as resolved but the changes haven't been made, any reason why?

Reply:
I have added the field in a new commit